apps/ui/lib/ui_web/live/price_feed_live.ex

```diff
@@ -55,9 +60,13 @@ defmodule UiWeb.PriceFeedLive do
     ticks = Hefty.fetch_streaming_symbols()
       |> symbols_to_keywords
 
-      ticks
-        |> Keyword.keys()
-        |> Enum.map(&(UiWeb.Endpoint.subscribe("stream-#{&1}")))
+    ticks
+      |> Keyword.keys()
+      |> Enum.map(&(UiWeb.Endpoint.subscribe("stream-#{&1}")))
+
+    atom = :null
```

Comment:
Could you please remove this and make `get_direction` take a single value but default to `\\ nil`
examples here: https://inquisitivedeveloper.com/lwm-elixir-25/